docs/manual/working/scalaGuide/main/sql/slick/PlaySlick.md

```diff
@@ -41,6 +41,9 @@ Note there is no need to add the Play `evolutions` component to your dependencie
 
 > **Note**: You can see th
 
+#### Running evolutions with Slick using compile-time DI
+If you are using Slick, you will also need to mix in the `SlickEvolutionsComponents` trait in addition to the `EvolutionsComponents` trait to `ApplicationComponents`. Else evolutions will not recognize the Slick database configurations in the configuration files.
+
```

Comment:
It could be good to add a code sample here. You can see the recommended way of adding code samples here:

https://playframework.com/documentation/2.5.x/Documentation#Code-samples